servicetests/tests/FleetProvisioning/fleet_provisioning.py

```diff
@@ -0,0 +1,448 @@
+    # type: (iotidentity.RejectedError) -> None
+    exit("CreateCertificateFromCsr Request rejected with code:'{}' message:'{}' status code:'{}'".format(
+        rejected.error_code, rejected.error_message, rejected.status_code))
+
+
+def registerthing_execution_accepted(response):
+    # type: (iotidentity.RegisterThingResponse) -> None
+    try:
+        global registerThingResponse
+        registerThingResponse = response
+        if (cmdData.input_is_ci == False):
+            print("Received a new message {} ".format(registerThingResponse))
+        return
+
+    except Exception as e:
+        exit(e)
+
+
+def registerthing_execution_rejected(rejected):
+    # type: (iotidentity.RejectedError) -> None
+    exit("RegisterThing Request rejected with code:'{}' message:'{}' status code:'{}'".format(
+        rejected.error_code, rejected.error_message, rejected.status_code))
+
+# Callback when connection is accidentally lost.
+def on_connection_interrupted(connection, error, **kwargs):
+    print("Connection interrupted. error: {}".format(error))
+
+
+# Callback when an interrupted connection is re-established.
+def on_connection_resumed(connection, return_code, session_present, **kwargs):
+    print("Connection resumed. return_code: {} session_present: {}".format(return_code, session_present))
+
+    if return_code == mqtt.ConnectReturnCode.ACCEPTED and not session_present:
+        print("Session did not persist. Resubscribing to existing topics...")
+        resubscribe_future, _ = connection.resubscribe_existing_topics()
+
+        # Cannot synchronously wait for resubscribe result because we're on the connection's event-loop thread,
+        # evaluate result with a callback instead.
+        resubscribe_future.add_done_callback(on_resubscribe_complete)
+
+
+def on_resubscribe_complete(resubscribe_future):
+    resubscribe_results = resubscribe_future.result()
+    print("Resubscribe results: {}".format(resubscribe_results))
+
+    for topic, qos in resubscribe_results['topics']:
+        if qos is None:
+            sys.exit("Server rejected resubscribe to topic: {}".format(topic))
+
+
+def waitForCreateKeysAndCertificateResponse():
+    # Wait for the response.
+    loopCount = 0
+    while loopCount < 10 and createKeysAndCertificateResponse is None:
+        if createKeysAndCertificateResponse is not None:
+            break
+        if not cmdData.input_is_ci:
+            print('Waiting... CreateKeysAndCertificateResponse: ' + json.dumps(createKeysAndCertificateResponse))
+        else:
+            print("Waiting... CreateKeysAndCertificateResponse: ...")
+        loopCount += 1
+        time.sleep(1)
+
+
+def waitForCreateCertificateFromCsrResponse():
+    # Wait for the response.
+    loopCount = 0
+    while loopCount < 10 and createCertificateFromCsrResponse is None:
+        if createCertificateFromCsrResponse is not None:
+            break
+        if not cmdData.input_is_ci:
```

Comment:
`input_is_ci` flag is redundant for this test. Also, I'd remove most of print statements as they don't provide any useful information for the test